cluster/gce/util.sh

```diff
@@ -2872,30 +2877,37 @@ http {
         set \$TENANT_API http://${RESOURCE_MASTER_IP}:8080;
 
         location ~ ^/[a-zA-Z0-9_.-]+$ {
-            proxy_pass \$RESOURCE_API;
+            proxy_read_timeout 3600;
+            proxy_pass http://\$remote_addr:8080;
         }
 
         location ~ ^/[a-zA-Z0-9_.-]+/[a-zA-Z0-9_.-]+$ {
-            proxy_pass \$RESOURCE_API;
+            proxy_read_timeout 3600;
+            proxy_pass http://\$remote_addr:8080;
         }
 
         location ~ ^/apis/[a-zA-Z0-9_.-]+/[a-zA-Z0-9_.-]+$ {
-            proxy_pass \$RESOURCE_API;
+            proxy_read_timeout 3600;
+            proxy_pass http://\$remote_addr:8080;
         }
 
         location ~ ^/api/v1/nodes?(.*) {
+            proxy_read_timeout 3600;
             proxy_pass \$RESOURCE_API;
         }
 
         location ~ ^/apis/coordination.k8s.io/v1/leases?(.*) {
```

Comment:
maybe incoorperate with Qian's PR 827 ?

Reply:
it doesn't work.

Reply:
there was bad merge earlier. after i rebased my changes on top of Qian's PR, I'm able to deploy the scaleout cluster now.